packages/ui/src/components/EventsCard/EventsCard.test.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { EventsCard } from './EventsCard';
+import { BUTTON_TEXT, events as listOfEvents, MAIN_TITLE } from './EventsCard.mocks';
+
+describe('EventsCard tests', () => {
+  describe('render component tests', () => {
+    it('main title should be render', () => {
+      // when
```

Comment:
🙃 Overall there is nothing bad in this practice, but we don't add such comments in other tests so you can remove yours for consistency